apps/browser/css/navbar.css

```diff
@@ -38,7 +38,14 @@
   border-radius: 3px;
 }
 
-.navbar > button.disabled {
+.urlbar > button {
+ background-image: url(images/shared/reload-stop-go.png);
+ background-size: 42px 28px;
+ width:12px;
+ height: 12px;
```

Comment:
They are in fact 12x12px. (The regular icons from toolbar.png are too large to fit in the urlbar, so these 12x12px ones are used instead)